Sources/KlaviyoUI/InAppForms/IAFWebViewModel.swift

```diff
@@ -26,6 +26,32 @@ class IAFWebViewModel: KlaviyoWebViewModeling {
     public let (navEventStream, navEventContinuation) = AsyncStream.makeStream(of: WKNavigationEvent.self)
     private let (formWillAppearStream, formWillAppearContinuation) = AsyncStream.makeStream(of: Void.self)
 
+    private var klaviyoJsWKScript: WKUserScript? {
+        guard let companyId = KlaviyoInternal.apiKey else {
+            if #available(iOS 14.0, *) {
+                Logger.webViewLogger.warning("Unable to initialize KlaviyoJS script on In-App Form HTML due to missing API key.")
```

Comment:
Does it make sense to emit a developer warning here like - https://github.com/klaviyo/klaviyo-swift-sdk/blob/e12f7c645db48dbf14264f5963d8d196631372a6/Sources/KlaviyoSwift/StateManagement/StateManagement.swift#L130